app/dashboard/templates/shared/yge_auth.html

```diff
@@ -28,30 +28,26 @@
   {{ user }}
 </div>
 {% else %}
-<script>
-  FontAwesomeConfig = { searchPseudoElements: true };
-</script>
-<script defer src="{% static "v2/js/fontawesome-all.min.js" %}"></script>
```

Comment:
Just curious. Don't you have to add the css link here as well like you did for `shared/head.html` ?

Reply:
Ew. I thought I added that